plugin-build/build.gradle.kts

```diff
@@ -62,27 +62,28 @@ dependencies {
 }
 
 configure<JavaPluginExtension> {
-    sourceCompatibility = JavaVersion.VERSION_1_8
-    targetCompatibility = JavaVersion.VERSION_1_8
+    toolchain {
+        languageVersion.set(JavaLanguageVersion.of(11))
```

Comment:
@marandaneto do you see any problems with that? We support AGP 7.0 min, which in turn supports Java 11 as min version, so I think we are good

Reply:
It should be fine.